docker/linux/base/Dockerfile

```diff
@@ -20,16 +20,21 @@ COPY base/clean-after-apt.sh /opt/clean-after-apt.sh
 
 ENV PYTHONUNBUFFERED 1
 
-# === Install common dependencies
-RUN apt update -qqy \
-    && apt install -qqy --no-install-recommends \
-        python3-dev \
-        python3-pip \
-        python3-requests \
-        python3-setuptools \
-        python3-six \
-        git ccache \
-        curl xz-utils \
-    && /opt/clean-after-apt.sh
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Every other container in this tree builds _on top_ of this container. Making global `ENV` changes could be done only here - other layers should not be needed, and should be backed out. Same for both `DEBIAN_FRONTEND` and `DEBCONF_NOWARNINGS` below

Second, there should actually be no need to set these vars - the image builds seem to work okay without it